pkg/config/middleware.go

```diff
@@ -53,5 +53,5 @@ type RequestLogger struct {
 
 type Auth struct {
 	Enabled bool     `json:"enabled"`
-	Tokens  []string `json:"-"`
+	Tokens  []string `json:"tokens"`
```

Comment:
This will result in all auth tokens being logged on startup instead of obfuscated. That should not happen.